src/main/java/openfasttrack/core/ChecksumCalculator.java

```diff
@@ -0,0 +1,41 @@
+package openfasttrack.core;
```

Comment:
Why is this in the `core` and not `importer`? Is it needed outside of the importer?

Reply:
:white_check_mark: